gradle/libs.versions.toml

```diff
@@ -232,7 +234,7 @@ kotlin-gradle-pluginApi = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin-
 kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-reflect", version.ref = "kotlin" }
 kotlin-stdlib-jdk8 = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk8", version.ref = "kotlin" }
 
-kotlin-compile-testing = "com.github.tschuchortdev:kotlin-compile-testing:1.4.9"
+kotlin-compile-testing = "dev.zacsweers.kctfork:core:0.2.1"
```

Comment:
`com.github.tschuchortdev:kotlin-compile-testing` hasn't been updated to support Kotlin 1.8 so I swapped it out for a forked version that has Kotlin 1.8 support.